simphony_metadata.yml

```diff
@@ -419,3 +419,28 @@ CUDS_KEYS:
   ELECTROSTATIC_MODEL:
     parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    physics_equation: [MOLECULAR_DYNAMICS]
+
+  BASIS:
+     definition: Space basis vectors
+     parent: CUBA.COMPONENT
+     CUBA.VECTOR:
+        shape: (3)
+        default: [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
+
+  ORIGIN:
+    definition: The origin of a space system
+    parent: CUDS:COMPONENT
+    CUBA.POINT:
+      default: [0 0 0]
+
+  BOX:
+    definition: A simple hexahedron box object
+    parent: CUBA.BOUNDARY
+    CUBA.BASIS:
+      default:
+        CUBA.VECTOR: [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
```

Comment:
In that case how can we recognize a default value for a primary CUBA from a default value for a structure with multiple CUBA values inside?

BTW, I think here I can remove the default all together, since it has the same value as in  BASIS itself.